onisite/test_settings.py

```diff
@@ -41,3 +41,13 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 )
+
+# Reset back to Django defaults to keep test output simple
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+    }
+}
+
+del LOGGING
```

Comment:
Tangential question for a possible new issue: If this line is removed there are some warnings and errors that come through in the testing logging. Should our tests give cleaner output if logging is enabled beyond out of the box Django logging?

Reply:
Created an issue for this: https://github.com/open-oni/open-oni/issues/567